### DISCUSSION TITLE
Spinners Having an "Acceleration" While Spinning.

### DISCUSSION THREAD

**Discussion by SevenEnd7** [ppy/osu#17246] [Ideas] [answered]
Right now on lazer, spinners have zero acceleration to them. You can go from 0 to 700+ RPM without much effort (making lines or even holding the pen in the center can achieve this too).
Adding some form of acceleration to the spinner will make it so you have to spin in one direction rather than moving clockwise/counterclockwise repeatedly to "spin". I can provide examples if anything is unclear, but if you go on lazer and start moving your cursor left and right on a spinner you can achieve 400-500 rpm with little effort. You'd be lucky to hit over 100 RPM doing the same thing on stable.

Side note: are there any plans to limit the RPM on spinners? Without any acceleration, spinners are trivial when it comes to maxing out the points you can get from them.

**Comment by peppy** [comment, marked as answer]
Please search first https://github.com/ppy/osu/discussions?discussions_q=spinner.
